# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderStateMachine.h
# Commit: 85ef72ebde43
# Full Hash: 85ef72ebde439c5de258af43faa9bd83a6e23b26
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-08-29 21:45:25
# Regressor Bug: 1573102
# File Overlap Count: 1
# Description:
#   Bug 1573102 - Remove mGraph from HTMLMediaElement::OutputMediaStream and OutputStreamManager. r=karlt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43666
# ==============================================================================

diff -r f61dd73cac3a -r 85ef72ebde43 dom/media/MediaDecoderStateMachine.h
--- a/dom/media/MediaDecoderStateMachine.h	Thu Aug 29 13:32:31 2019 +0000
+++ b/dom/media/MediaDecoderStateMachine.h	Thu Aug 29 13:32:50 2019 +0000
@@ -188,7 +188,7 @@
 
   void SetOutputStreamPrincipal(nsIPrincipal* aPrincipal);
   // If an OutputStreamManager does not exist, one will be created.
-  void EnsureOutputStreamManager(MediaStreamGraphImpl* aGraph);
+  void EnsureOutputStreamManager(MediaStreamGraph* aGraph);
   // If an OutputStreamManager exists, tracks matching aLoadedInfo will be
   // created unless they already exist in the manager.
   void EnsureOutputStreamManagerHasTracks(const MediaInfo& aLoadedInfo);